DeploymentCloud/Deployment.Common/Resources/Parameters/Spark-AutoScale-Parameter.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Why create a new parameter file. Should we re-use the existing Spark-Parameter.json file and add the new parameters to it ?

Reply:
I saw each template json file has its own parameter file under .\DeploymentCloud\Deployment.Common\Resources\ folder, so I created this separate parameter file for the new template file. You are correct. We should be able to use the same spark-parameter.json parameter file for both templates (Spark-template.json and Spark-AutoScale-Template.json).

Reply:
removed the spark-autoscale-parameter.json file